File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
 int majorityElement(vector<int>& nums) {
    map<int,int>mp;
    for (int i = 0; i < nums.size(); ++i) {
        mp[nums[i]]++;
    }
    int m{};
    for(auto i:mp ){
        if(i.second>nums.size()/2){
            m=i.first;
            break;
        }
    }
    return m;
     
}
};

File: 0169-majority-element/0169-majority-element.cs
public class Solution {
public  int MajorityElement(int[] nums)
{
	var result = new Dictionary<int, int>();

	for (int i = 0; i < nums.Length; i++)
	{
		if (result.ContainsKey(nums[i]))
			result[nums[i]]++;
		else
		{
			result.Add(nums[i], 1);
		}

	}

	return result.Where(x => x.Value >( nums.Count() / 2)).FirstOrDefault().Key;
}
}